src/WebJobs.Extensions.DurableTask/HttpApiHandler.cs

```diff
@@ -322,43 +333,44 @@ public async Task<HttpResponseMessage> HandleRequestAsync(HttpRequestMessage req
         {
             IDurableOrchestrationClient client = this.GetClient(request);
             var queryNameValuePairs = request.GetQueryNameValuePairs();
-            var createdTimeFrom = GetDateTimeQueryParameterValue(queryNameValuePairs, CreatedTimeFromParameter, default(DateTime));
-            var createdTimeTo = GetDateTimeQueryParameterValue(queryNameValuePairs, CreatedTimeToParameter, default(DateTime));
-            var runtimeStatus = GetIEnumerableQueryParameterValue<OrchestrationRuntimeStatus>(queryNameValuePairs, RuntimeStatusParameter);
-            var pageSize = GetIntQueryParameterValue(queryNameValuePairs, PageSizeParameter);
 
-            var continuationToken = "";
-            if (request.Headers.TryGetValues("x-ms-continuation-token", out var headerValues))
+            var condition = new OrchestrationStatusQueryCondition();
+
+            if (TryGetDateTimeQueryParameterValue(queryNameValuePairs, CreatedTimeFromParameter, out DateTime createdTimeFrom))

```

Comment:
In general, I feel the move from `Get*Value()` method with defaults to `TryGet*Value()` methods made the code more clunky, without gaining much. We should consider moving back to the old approach.

Reply:
I actually suggested that Amanda do it this way. Otherwise we're always overwriting the default values in the `OrchestrationStatusQueryCondition` object with some other defaulting behavio, which isn't ideal IMO. I see the `TryGet` approach as minimizing duplication.